WordPress/Classes/ViewRelated/Post/PostSettings/PostSettingsView.swift

```diff
@@ -267,9 +267,13 @@ struct PostSettingsFormContentView: View {
 
     private var statusRow: some View {
         NavigationLink {
-            PostStatusView(settings: $viewModel.settings, timeZone: viewModel.timeZone)
+            PostStatusView(
+                settings: $viewModel.settings,
+                timeZone: viewModel.timeZone,
+                isPublishing: viewModel.context == .publishing
+            )
         } label: {
-            SettingsRow(Strings.status) {
+            SettingsRow(viewModel.context == .publishing ? Strings.statusAndVisibility : Strings.status) {
```

Comment:
`PostStatusView` title is always "Status & Visibility", but why use a different text for the entrance in the list?

Reply:
I guess let's just call it "Status" like on the web. "Status" is a quintessential WordPress post thing. There is no such thing as "Visibility".